### ISSUE TITLE
Update README and DESCRIPTION

### ISSUE THREAD

**Issue by pbiecek** [ModelOriented/DALEXtra#20]
After #19 it would be good to update README and DESCRIPTION in order to reflect new content of the package

In addition to support for mlr, h2o and scikit-learn we also need to mention champion-challenger funnel plots and aspect importance features.

**Comment by maksymiuks** [comment]
Thanks, I've planned to take care about it this weekend.

**Comment by pbiecek** [comment]
thanks!

**Comment by maksymiuks** [comment]
Merged into master:)
